### DISCUSSION TITLE
NBMiner v39.7

### DISCUSSION THREAD

**Discussion by nebutech-admin** [NebuTech/NBMiner#568] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/NebuTech/NBMiner/releases/tag/v39.7'>NBMiner v39.7</a>.</em>

**Comment by KaHDaPaIII** [comment]


**Reply by jefersons2004** [reply]
I have the same problem when I use different version of the NVidia driver, in my case it only works with the 460.84 driver, my pc is an amd2 "N68-S3 FX ASRock (P1.20 06/04/2012)"

**Reply by jefersons2004** [reply]
test the first driver that works and return one to see if the MH/s improves....

**Reply by Faltskog** [reply]
I get the impression that the v39.6 worked better and stable...
I was getting constant 88~89mh/s with v39.6, now with 39.7 i can't pass 86, same settings.
I will rollback to previous ver to check that out.

**Reply by ZHNovell** [reply]
i have the same problem 3060 LHR V2 driver 472.12  , is there a solution?

**Reply by alexsandrn** [reply]
I think the problem will be closed in the next version of the miner

**Reply by dayoocn11** [reply]
After 3060ti runs for a period of time, the computing power suddenly drops from 43m to 22m

**Comment by Faltskog** [comment]
I get the impression that the v39.6 worked better and stable...
I was getting constant 88~89mh/s with v39.6, now with 39.7 i can't pass 86, same settings.
I will rollback to previous ver to check that out.

GPU 3080ti

**Reply by Vixus** [reply]
I feel the same too with my 3080ti. The other thing I don’t understand is that when I increase my GPU clock, I get significantly reduced hash rate and yet the LHR doesn’t trigger. It’s hard for me to optimize because I don’t understand how increasing mem clock could decrease hash rate without hitting the lock. I have mine set to mem clock +1250MHz. Anything above this and it drops to 60MH.

**Reply by Faltskog** [reply]
> I feel the same too with my 3080ti. The other thing I don’t understand is that when I increase my GPU clock, I get significantly reduced hash rate and yet the LHR doesn’t trigger. It’s hard for me to optimize because I don’t understand how increasing mem clock could decrease hash rate without hitting the lock. I have mine set to mem clock +1250MHz. Anything above this and it drops to 60MH.

Yesterday i corroborate the exact same thing with my Gigabyte VISION 3080ti OC, anything above 1250+ mem clock will drop out to 70, i think that's the sweet spot.

At least, for the primary device, i have 2 exactly same 3080ti and the secundary (device 1) i can push it a little further, maybe is because is not using power to boot up windows and it's processes and stuff like that, i don't know, could it be silicon lottery but i can push it to 1400mc and get around 90~91mh/s. Try that out.
The primary one tho (device 0) goes with the same rule as you said, no more than 1250mc

The thing is, i know people that can achieve 220w power consumption, and my stats are as follows:

90mhs 286w 55CTmp 84MTmp 70Fan (fixed) 1300 CClk 10451 GMClk (getting above 305~315 kH/w)

**Reply by Vixus** [reply]
Something strange is happening. I noticed that sometimes my hash rate is super low (low 80 to high 70), but after playing a game, if I return to mining I can sometimes hit 89-90MH and even bump my lhr to 75.5. Not sure what’s triggering it but so far looking good.

**Reply by R92H** [reply]
> Something strange is happening. I noticed that sometimes my hash rate is super low (low 80 to high 70), but after playing a game, if I return to mining I can sometimes hit 89-90MH and even bump my lhr to 75.5. Not sure what’s triggering it but so far looking good.

For which version is that ? Also  does any of you guys get LHR limiter to kick in after couple of hours ?

**Comment by lexus440** [comment]
What driver is recommended on linux(hiveos) for this v39.7 mining with 3060 LHRv1 and 3090?

Now I run on 495.44, but I know 460.39 is recommended for another miner.

[14:30:13] INFO - |ID|Device|Hashrate|Accept|Reject|Inv|Powr|CTmp|MTmp|Fan|CClk|GMClk|MUtl|Eff/Watt|                                          
[14:30:13] INFO - | 0|  3060| 34.36 M|    32|     0|  0| 110|  68|    | 58|1605| 8200| 100| 312.4 K|                                          
[14:30:13] INFO - | 1|  3060| 34.38 M|    40|     0|  0| 110|  60|    | 31|1605| 8200| 100| 312.5 K|                                          
[14:30:13] INFO - | 2|  3090| 119.0 M|   101|     0|  0| 299|  56|    | 92|1305|10451| 100| 398.1 K|                                          
[14:30:13] INFO - | 3|  3090| 65.19 M|    64|     0|  0| 225|  53|    |100| 585|10451|  66| 289.7 K|                                          
[14:30:13] INFO - | 4|  3090| 111.2 M|   110|     0|  0| 299|  60|    | 89|1035|10451| 100| 371.8 K|                                          
[14:30:13] INFO - |------------------+------+------+---+----+--------------------------------------|                                          
[14:30:13] INFO - |    Total: 364.1 M|   347|     0|  0|1043| Uptime:  0D 01:14:30        CPU:  5% |  

(This v39.7 NBMiner is the first to run stable on this rig.)

**Comment by Kafila** [comment]
Is there anything i need to do to get better with my v1 3060's 
I use nicehash... updated to nbminer 39.7 and just started it gave 2 rejects in one card and 1 on the other... just in the start... 
I havent found stability yet and my miner restarts after some hours... from 8 to 15... 
![16356995032442691016040058485780](https://user-images.githubusercontent.com/23731101/139594144-2db7756e-483d-42f5-b1bd-8229b2555a76.jpg)

**Reply by Ninjapods** [reply]
Lower your OC settings. I realized that the more GPU cards you has the OC have to be lowered.
For example: When I had 2x 3060 1x 2080 Super - I could run them up to Mem +1500 now that I have 4x3060 1x2080 Super
The 3060 can run max on Mem +1383 and the super Mem +1460 (Core Clock -420, -400 depends on your Silicon Lotery)

Also you have to increase (Custom Size) your virtual memory - The virtual memory needed is reciprocal to the GB of your cards + 4GB for your OS. (https://www.nicehash.com/blog/post/how-to-increase-virtual-memory-on-windows?utm_source=PDF&utm_medium=Troubleshooting)

**Reply by ZHNovell** [reply]
Thank you. I have the same problem, the advice didn’t help. OC settings - default, virtual memory - Custom Size 25 GB. Other miners work well, but NB doesn't want to ...
[22:37:09] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
[22:37:09] ERROR - Mining program unexpected exit.
[22:37:09] ERROR - Code: -1073741676, Reason: Process crashed
[22:37:09] ERROR - Restart miner after 10 secs ...
[22:37:09] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 
3060 LHR V2 Nvidia driver 496.49 changed for 412.72  did not help.

**Reply by Kafila** [reply]
I did it's working for now more than 1 hour 0 rejects... I think there were 3 or 4 issues 
1 internet conection... now is a better one 
2 clocks to high... I will try a bit higher... but good for now... 
3 virtual memory and disabling power and device status monitoring on nice hash... as it's known to consume memory...
And 4 i didn't get it solved but i think my power is unstable... as my light trembles... waiting for a ups to see final results but for now great...

**Reply by Kafila** [reply]
> Thank you. I have the same problem, the advice didn’t help. OC settings - default, virtual memory - Custom Size 25 GB. Other miners work well, but NB doesn't want to ...
> [22:37:09] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
> [22:37:09] ERROR - Mining program unexpected exit.
> [22:37:09] ERROR - Code: -1073741676, Reason: Process crashed
> [22:37:09] ERROR - Restart miner after 10 secs ...
> [22:37:09] ERROR - !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 
> 3060 LHR V2 Nvidia driver 496.49 changed for 412.72  did not help.

How do you get that log??? Are you using nicehash??? Cause i can get it out of nicehash but with nbminer on nicehash... I cant find it
THANKS

**Reply by Kafila** [reply]
> Lower your OC settings. I realized that the more GPU cards you has the OC have to be lowered.
> For example: When I had 2x 3060 1x 2080 Super - I could run them up to Mem +1500 now that I have 4x3060 1x2080 Super
> The 3060 can run max on Mem +1383 and the super Mem +1460 (Core Clock -420, -400 depends on your Silicon Lotery)
> 
> Also you have to increase (Custom Size) your virtual memory - The virtual memory needed is reciprocal to the GB of your cards + 4GB for your OS. (https://www.nicehash.com/blog/post/how-to-increase-virtual-memory-on-windows?utm_source=PDF&utm_medium=Troubleshooting)
> 

Mine are runing at 1180 and cant go much further... only if you give them power... and that will not be sustainable... I will give you my temps and results for first hour as well as oc tune values... 

I never had the miner runing for more than 12 hours...
 
Afterburner

Card 0 tu104-a 2070super 
Memory samsung

PL 46 
TL 72°
Cclck -105
Mclck 999
Fan 85

Card 1 asus 3060 ga106 lhr a1 
Memory samsung 
PL 64
TL 72 
Cclck -502
Mclck 1180 
Fan 90

Card 2 gigabyte 
PL 63 
TL 72°
Cclck -502
Mclck 1180 
Fan 91![20211102_055615.jpg](https://user-images.githubusercontent.com/23731101/139793862-a0078ae9-bcc9-46ee-ae95-ea17c55bcfc4.jpg)

**Reply by Ninjapods** [reply]
@Kafila Does the issue still persists?
I have an upate about my Rev 1 RTX 3060's. I had to lower all the OC settings even more.

Gigabyte Brand: -400, +1340, 70% (49.34MHs) 118W
Rogstrix Tuff brand: -400, +1298 to +1340, 70% (49.35-49.70) 118W

It has been running for 48hrs with no issues so far (no NBminer restarts) and only 2 rejects on one card.

**Reply by Kafila** [reply]
You still running like that... I had to adjust stuf arroun and coudnt get the miner in place agai  but one 2 days ready I hope i have same clocks runing now.
No rejects till now but very recent...

**Comment by tsc2017** [comment]
What hashrate will I get if I don't use the 470.05 driver for RTX 3060v1?

**Reply by Ninjapods** [reply]
I have an update about my Rev 1 RTX 3060's. I had to lower all the OC settings even more.

Gigabyte Brand: -400, +1340, 70% (49.34MHs) 118W
Rog-strix Tuff brand: -400, +1298 to +1340, 70% (49.35-49.70) 118W

It has been running for 48hrs with no issues so far (no NBminer restarts) and only 2 rejects on one card.

**Comment by ImpliedConsent** [comment]
NBMiner_39.7_Win.zip is failing. 
https://github.com/NebuTech/NBMiner/releases/download/v39.7/NBMiner_39.7_Win.zip

![Screenshot 2021-11-01 095621](https://user-images.githubusercontent.com/29551816/139701715-89074b5a-0ce7-47b0-a721-40032c56bb3b.jpg)

**Reply by moops44** [reply]
Maybe ur Antivirus app, if not in Chrome open the Downloads Tab and retry the download

**Comment by alexsandrn** [comment]
how to turn on  -lhr-mode 2?
-71 for my 3060Ti LHR is a lot

**Reply by papirdos** [reply]
how to install LHR 69 ???

**Comment by R92H** [comment]
Hi , single 3080Ti here locked at 1400MHz / 725mV for the core and 10500 MHz ( +1250) for the memory im getting 88mhs at 230-240W  for couple of  hours ( 10-12h ) then LHR limiter seems to kick in and the hashrate drop to 44mhs or something , so im forced to restart the rig .

My question is , isn't v39.7 supposed to fix that issue ?

**Reply by rafamonteiroo** [reply]
> Hi , single 3080Ti here locked at 1400MHz / 725mV for the core and 10500 MHz ( +1250) for the memory im getting 88mhs at 230-240W for couple of hours ( 10-12h ) then LHR limiter seems to kick in and the hashrate drop to 44mhs or something , so im forced to restart the rig .
> 
> My question is , isn't v39.7 supposed to fix that issue ?

I have this same problem with the 3080ti, nbminer does not detect that it blocked LHR

**Reply by Ahm3dAG** [reply]
rolll back your drivers, new drivers bad for mining

**Reply by rafamonteiroo** [reply]
> rolll back your drivers, new drivers bad for mining

Which version do you recommend? I'm using 496.49... version 472.12 doesn't display consumption on nbmimer

**Reply by smarley2** [reply]
I´m using 460.91.03 Nvidia Driver

**Reply by Ahm3dAG** [reply]
Im using 472.12, what consumption do you mean? and yea 496.69 is DCH and a lot of peoples hash rate just drops after a few minutes, non lhr and lhr cards

**Reply by Ahm3dAG** [reply]
> 
> 
> > rolll back your drivers, new drivers bad for mining
> 
> Which version do you recommend? I'm using 496.49... version 472.12 doesn't display consumption on nbmimer

Anything before 496.69 i think, they stopped releasing non DCH drivers now

**Reply by rafamonteiroo** [reply]
> Im using 472.12, what consumption do you mean? and yea 496.69 is DCH and a lot of peoples hash rate just drops after a few minutes, non lhr and lhr cards

I installed version 466.77 standard, non dch.... this version showed the consumption of the 3070ti and 3080ti, but the 1070 didn't show...

I remember the 472.12 driver didn't show any... does anyone know how to fix this?

https://i.imgur.com/r0gQxzA.png

**Comment by Kafila** [comment]
Yeah  in this one i think... my is very nice now... had to low down my 2070s now I only get 40 mh out of it but all is stable...
I will try to get more out of her latter... or maybe now hahahah

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Kafila** [comment]
Daggerhashimoto

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by papirdos** [comment]
how to install LHR 69 ???

**Comment by Kafila** [comment]
To work with that you have to use the normal driver... but it will run the card lower than with the juicy driver...

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by smarley2** [comment]
How can I use the orther arguments?
I try this bellow but did not work the lhr-reduce-value, time and limit.
Just "lhr" is  working.

"platform": 1
"lhr": "75, 75, 75"
"lhr-reduce-value": "0.1, 0.1, 0.1"
"lhr-reduce-time": "10, 10, 10"
"lhr-reduce-limit": "10, 10, 10"

**Comment by x77xx96** [comment]
каждые 10 -15 мин вылетает ошибка 
![Снимок экрана (137)](https://user-images.githubusercontent.com/15983071/140660816-c9f379f9-3832-47cb-b61c-f4e81e5368fa.png)

**Reply by Kafila** [reply]
In eussian all i kan say is vodka... but clocks are too high... or temperature limits too low...

**Reply by Sunny7860101** [reply]
Mine 3080ti pl70 cc-150, mc1350, f80 =90mh/s

**Comment by Kafila** [comment]
Try add your lhr to the config line... I think you can and it will force it...

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by powpppp** [comment]
powpat547@gmail.com
0969423416

**Comment by dayoocn11** [comment]
472.12 yes, now back to 39.6, stable operation
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NebuTech/NBMiner"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月11日(星期四) 凌晨3:16
***@***.***&gt;;
抄送:&nbsp;"德 善 ***@***.******@***.***&gt;;
主题:&nbsp;Re: [NebuTech/NBMiner] NBMiner v39.7 (Discussion #568)





  
Im using 472.12, what consumption do you mean? and yea 496.69 is DCH and a lot of peoples hash rate just drops after a few minutes, non lhr and lhr cards
  
this pwer
 
[img]https://i.imgur.com/oeWwsbR.png[/img]
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by dayoocn11** [comment]
472.12 yes, now back to 39.6, stable operation
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NebuTech/NBMiner"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月11日(星期四) 凌晨2:47
***@***.***&gt;;
抄送:&nbsp;"德 善 ***@***.******@***.***&gt;;
主题:&nbsp;Re: [NebuTech/NBMiner] NBMiner v39.7 (Discussion #568)





   
rolll back your drivers, new drivers bad for mining
  
Which version do you recommend? I'm using 496.49... version 472.12 doesn't display consumption on nbmimer
  
Anything before 496.69 i think, they stopped releasing non DCH drivers now
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by dayoocn11** [comment]
472.12 yes, now back to 39.6, stable operation


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NebuTech/NBMiner"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年11月11日(星期四) 凌晨2:46
***@***.***&gt;;
抄送:&nbsp;"德 善 ***@***.******@***.***&gt;;
主题:&nbsp;Re: [NebuTech/NBMiner] NBMiner v39.7 (Discussion #568)





 
Im using 472.12, what consumption do you mean? and yea 496.69 is DCH and a lot of peoples hash rate just drops after a few minutes, non lhr and lhr cards
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by dayoocn11** [comment]
472.12 yes, now back to 39.6, stable operation




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年11月11日(星期四) 凌晨2:20
收件人: ***@***.***&gt;; 
抄送: "德 善 ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [NebuTech/NBMiner] NBMiner v39.7 (Discussion #568)






回滚你的驱动，新的驱动不适合挖矿

—
你收到这个是因为你评论了。
直接回复此邮件，在 GitHub 上查看或取消订阅.
GitHub Mobile 上的分流通知监督办或机器人.

**Comment by Hmmsultan** [comment]
Hi.
who can help.
what are the optimal parameters MH/S for rtx 3060 ghost oc Gainward rev2 Samsung.
which  you use
NB,LOL...
![20211113_220525](https://user-images.githubusercontent.com/94250185/141657594-d40a23af-f596-4f88-b202-a65210c7620c.jpg)

**Comment by pattrann** [comment]
quick question, does this miner use opengl? I'm able to run in using NVIDIA, but doesn't work with intel gpu. Since NVIDIA is too powerful for etchash algorithm i'm using intel UHD 620 gpu, any help greatly appreciated.

**Comment by Kafila** [comment]
Youuu don have any intel gpu... what you got is an integrated graphics unit in the intel processor that sucks ram out... your ram... so no...

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by pattrann** [reply]
i see the problem, is there a way to get it running on a 2gb NVIDIA mx130?

**Comment by Kafila** [comment]
I don't think so and if you could it give a what???

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by mj-malabi** [comment]
Im receiving error. What should i do please? 

<img width="670" alt="image" src="https://user-images.githubusercontent.com/101132997/157088241-286ebd38-541f-4bbd-b154-be1710f716b7.png">

**Reply by Kafila** [reply]
That card will not be profitable... it can make 3 dollars month... 
Do not put it to mine...

**Comment by Tom5051** [comment]
your card does not have enough gpu memory to load the dag for Ethereum mining. There is nothing you can do about it. You need a card with 6GB or more.
